internal/engine/inputloader.go

```diff
@@ -147,6 +162,66 @@ func (il *InputLoader) loadOrQueryBackend(ctx context.Context) ([]model.URLInfo,
 	return il.loadRemote(ctx)
 }
 
+// TODO(https://github.com/ooni/probe/issues/1390): we need to
+// implement serving DNSCheck targets from the API
+var dnsCheckDefaultInput = []string{
+	"https://dns.google/dns-query",
+	"https://8.8.8.8/dns-query",
+	"dot://8.8.8.8:853/",
+	"dot://8.8.4.4:853/",
+	"https://8.8.4.4/dns-query",
+	"https://cloudflare-dns.com/dns-query",
+	"https://1.1.1.1/dns-query",
+	"https://1.0.0.1/dns-query",
+	"dot://1.1.1.1:853/",
+	"dot://1.0.0.1:853/",
+	"https://dns.quad9.net/dns-query",
+	"https://9.9.9.9/dns-query",
+	"dot://9.9.9.9:853/",
+	"dot://dns.quad9.net/",
+}
+
+var stunReachabilityDefaultInput = stuninput.AsnStunReachabilityInput()
+
+var exampleWithDefaultInputInput = []string{
+	"https://example.com",
+	"https://example.org",
+}
+
+// StaticBareInputForExperiment returns the list of strings an
+// experiment should use as static input. In case there is no
+// static input for this experiment, we return an error.
+func StaticBareInputForExperiment(name string) ([]string, error) {
```

Comment:
This function is public because `pkg/oonimkall` does not use `InputLoader` directly for loading inputs. There is need to refactor how we run experiments on mobile. (やれやれ)